test/cloak/vault_test.exs

```diff
@@ -90,13 +103,64 @@ defmodule Cloak.VaultTest do
       assert ciphertext != "plaintext"
     end
 
+    test "encrypts ciphertext with default padding" do
+      assert ciphertext = TestVault.encrypt!("plaintext", padding: true)
+      assert is_binary(ciphertext)
+      assert ciphertext != "plaintext"
+    end
+
+    test "encrypts ciphertext with custom padding" do
+      assert ciphertext = TestVault.encrypt!("plaintext", padding: 24)
+      assert is_binary(ciphertext)
```

Comment:
This test is pretty much identical to the test above, so all it tests is that the `padding: 24` option is accepted, not that it actually pads up to 24 bytes. I think we should test that the ciphertext is 24 bytes long, if possible.

Reply:
I agree! And also wondered about this while creating this text. What should be the length of the encrypted text when not taking padding into account?